src/mutations/WaiverdbNew.js

```diff
@@ -1,20 +1,40 @@
+/*
+ * This file is part of ciboard
+
+ * Copyright (c) 2021, 2022 Andrei Stepanov <astepano@redhat.com>
+ * 
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
 import { gql } from '@apollo/client';
 
 export default gql`
     mutation WaiverdbNew(
-        $subject_type: String!,
```

Comment:
@mgrabovsky I think we have an issue here :-(
It was auto-save issue when prettify added commas.
I understand when re-formatting was done intentionally, and changed like you had before.
With different versions of prettify there can be minor changes, this is one of it.
Do you refuse to ACK because of this?

Reply:
This is my weak preference, but i would be nice to split these changes into a separate commit at least.

Reply:
How?
Auto-reformating works only on saved files.
How do I know whata files I will change in advance?

Reply:
If you're using VS Code, you can use the [built-in Git support](https://code.visualstudio.com/Docs/editor/versioncontrol) to inspect which lines have changed and pick individual files or even chunks of code within a file to the stage.

Reply:
I like Matej's suggestion here. Generally on projects I worked on, we should not bundle unrelated changes together with the change the MR was about. You can see this logic in most of the projects which have some "submitting patches" guidelines, for example https://www.kernel.org/doc/html/v4.12/process/submitting-patches.html#separate-your-changes or https://git-scm.com/docs/SubmittingPatches#separate-commits.

But in this single line case, I would say this is not a blocker, but it would be nice to change the workflow in a way that the code which is not needed in a MR is not part of MR.

Reply:
> I like Matej's suggestion here. Generally on projects I worked on, we should not bundle unrelated changes together with the change the MR was about. 

Yes. @jankaluza  where do I not agree with this?

Reply:
@mgrabovsky will send me a link how this can be improved in the feature. Resolving for now.